languages/java11/template/src/main/java/com/algorithmia/algorithm/Algorithm.java

```diff
@@ -0,0 +1,35 @@
+package com.algorithmia.algorithm;
+
+import com.algorithmia.development.*;
+import com.algorithmia.*;
+
+import java.util.HashMap;
+
+
+// This class defines your algorithm.
+class Algorithm extends AbstractAlgorithm<String, String>{
+
+    // This class member can be defined as anything you wish, and stores loaded state and mutable state between requests.
+     HashMap<String, String> context = new HashMap<>();
+
+    // This apply function defines the primary motive driver of your algorithm. Please ensure that the types defined in
+    // your algorithm are the same as those defined in your class implementation above ^
+
+    public String apply(String input){
+        // ... = context.get("loaded_model_file")
+        return "hello ".concat(input);
+    }
+
+    // For more information around how loading works, please check out the algorithm docs in algorithmia.com/developers/
+    //<--- Implement this for loading functionality -->
+    public void load(){
```

Comment:
Same general question: do we really need `load` in the base algorithm template. It seems super useful in a template for "Java with <Framework>" where the template shows how to load a model for that framework. I'm not sure if we have any statistics on what percentage of Java algorithms need a loading step. If it's a low percentage, then I think the extra code and comments are noise, instead of signal.